.pipelines/build-pipelines.yml

```diff
@@ -49,6 +50,12 @@ steps:
     echo "##vso[task.setvariable variable=dabVersion]$dabVersion"
   displayName: Set dab version
 
+- task: CmdLine@2
+  displayName: 'Set flag to publish received files when previous step fails'
```

Comment:
which recieved files are we mentioning here? can you please give an example?

Reply:
For Verify tests, when they fail they generated a `.received` file, meaning "this is what was received by Verify, not what has been verified".

Updated the display name to be clearer.